modules/apps/foundation/frontend-theme/theme3/src/css/_custom.scss

```diff
@@ -0,0 +1,335 @@
+                .table-striped > tbody > tr:nth-child(odd) {
+                    color: #000
+                }
+            }
+        }
+        
+        #banner {
+            background: url('../images/pine_forest_in_estonia.jpg') center/cover no-repeat fixed;
+            height: 1000px;
+            width: 100%;
+        }
+
+        #navigation {
+            background: linear-gradient(rgba(45, 70, 76, 1), rgba(45, 70, 76, 0.1));
+            height: 105px;
+            position: fixed;
+            width: 100%;
+            z-index: 99;
+
+            /*--------------DesktopNav--------------*/
+
+            @include media-query($screen-md-min, $screen-lg + 2000) {
+
+                &:hover {
+                    height: 125px;
+                }
+
+                &.hidenav  {
+                    top: -5px;
+        
+                    &:hover {
+                        height: 140px;
+                        top: 65px;
+                    }
+                }
+            }
+
+            /*------MobileNav------*/
+
+            @include media-query(null, $screen-xs) {
+                height: 470px;
+                padding: 10px 14px;
+                width: 70px;
+
+                ul {
+                    display: block;
+                    font-size: 12px;
+                    position: fixed;
+                    right: 395px;
+                    width: 20px;
+
+                    .child-menu {
+                        visibility: hidden;
+                    }
+
+                    li a {
+                        height: 1px;
+                        display: grid;
+                        width: 116px;
+                    }
+                }    
+
+                &.hidemobilenav {
+                    right: 368px;
+                    transition-property: right;
+                    transition-duration: .5s;
+
+                    li {
+                        visibility: hidden;
+                    }
+
+                    .child-menu {
+                        visibility: hidden;
+                    }
+
+                    &.togglesmobilenav {
+                        right: 314px;
+                        
+                        li {
+                            visibility: visible;
+                        }
+
+                        .child-menu {
+                            visibility: visible;
+                        }
+                    }
+                }
+            }    
+
+            /*------TabletNav------*/
+
+            @include media-query(($screen-xs-max - 30px), $screen-sm-max) {
+                position: fixed;
+                top: auto !important;
+
+                &.hidenav  {
+                    top: -5px !important;
+
+                    &.togglesnav {
+                        height: 120px;
+                        top: -15px;
+                    }
+                }    
+            }
+                
+            .active {
+                background-color:#0A1B20;
```

Comment:
Missing space between CSS property and value.  Please see https://github.com/jonmak08/liferay-portal/pull/1835#discussion_r167381758 for more details.